### ISSUE TITLE
Gem::LoadError on Vagrant provisioning

### ISSUE THREAD

**Issue by neeraj-ugam** [geerlingguy/drupal-vm#2080]
### Issue Type

 - Bug Report / Support Request
 
### Your Environment

```
Vagrant 2.2.6
6.1.10_Ubuntur138449
ansible 2.9.6

```

### Your OS

 - **Ubuntu** (20.04.1 LTS)

### Full console output

OptiPlex-3040:/var/www/html/drupalvm$ vagrant up
Installing plugin vagrant-hostsupdater
Vagrant failed to initialize at a very early stage:

There was an error loading a Vagrantfile. The file being loaded
and the error message are shown below. This is usually caused by
a syntax error.

Path: /var/www/html/drupalvm/Vagrantfile
Line number: 0
Message: Gem::LoadError: can't activate vagrant-libvirt-0.1.2, already activated vagrant-libvirt-0.0.45
OptiPlex-3040:/var/www/html/drupalvm$ 

### Summary

- While running "vagrant up" it throws the above error regarding presence of an enhanced version of vagrant-libvirt.
- Tried running "vagrant up --provision" with the same result.
- Uninstalling and reinstalling vagrant was of no use

**Comment by oxyc** [comment]
Most likely not drupal-vm but `vagrant-hostsupdater` an older upstream issue mentions it works if they uninstalled `vagrant-libvirt` (with apt) and then installed `vagrant-hostsupdater` which installs a newer version. https://github.com/agiledivider/vagrant-hostsupdater/issues/169

**Comment by neeraj-ugam** [comment]
I've given up on Ubuntu and now trying to get it running on a Mac. Now facing another problem: https://github.com/geerlingguy/drupal-vm/issues/2022#issuecomment-689535182

@geerlingguy : Help please.

P.S.: Closing this one as it's no longer needed.

**Comment by 216ann** [comment]
I have been trying to get a drupalVM to work that was on my old 7200 drive.  

My problem is very similar to
Gem::LoadError: can’t activate vagrant-libvirt-0.2.1, already activated vagrant-libvirt-0.0.45

and somewhat similar to Gem::LoadError: can’t activate vagrant-libvirt-0.1.2, already activated vagrant-libvirt-0.0.45

I moved my 7200HD (which has a drupalVM website) to another computer but was unable to boot it.
I was able to attach another 5400HD and use a live USB to format that into Ubuntu 20 LTS
Using that 5400HD, I was able to rsync all (at least I think) the files/directories from the 7200HD to the 5400HD.
To double check, I tried to vagrant up using the 5400HD copy to try to bring up the drupalVM
That returns: 
**Gem::LoadError: can’t activate vagrant-libvirt-0.3.0, already activated vagrant-libvirt-0.0.45** 

I had not installed vagrant/virtual box to the fresh 5400HD install so I did that but the same error returns
I tried removing as well as reinstalling as discussed by fcojgodoy but that was not successful.
Any ideas would be appreciated.
I already tried the Vagrant Community Forum but noone has responded in almost 2 weeks.  
Thanks for your help.
